### ISSUE TITLE
Figure out how to get a JSON file you can just upload to Jaeger UI

### ISSUE THREAD

**Issue by amoeba** [amoeba/arrow-opentelemetry-example#1]

**Comment by amoeba** [comment]
I found this blog post https://www.honeycomb.io/blog/test-span-opentelemetry-collector/ that showed an example. This looks like the JSON-encoded (versus binary) version of a protobuf schema. This hangs the Jaeger UI so far so I'm not sure what's going on.

**Comment by amoeba** [comment]
Turns out it's just an ad-hoc (non standard) JSON data format defined only in Jaeger UI. I found this note in https://github.com/jaegertracing/jaeger-ui/issues/381 which led me to find that the JSON file should look like:

```json
{
    "data": [
        {
            "traceID": "665b3cc06e48f122",
            "spans": [
                {
```

which matches this TS type: https://github.com/jaegertracing/jaeger-ui/blob/main/packages/jaeger-ui/src/types/trace.tsx#L72.

The structure is fairly similar to OpenTelemetry's JSON/Protobuf structure but different enough it doesn't seem worth doing some kinda of transformation.
